trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/is-pdc-reconcile-user.ts

```diff
@@ -0,0 +1,10 @@
+import { TEAM_IDS } from '@ukef/dtfs2-common';
+import { TfmSessionUser } from '../../../types/tfm-session-user';
+
+/**
+ * isPDCReconcileUser
+ * returns true if user is in PDC_RECONCILE team
+ * @param {TfmSessionUser} user
+ * @returns {Boolean}
```

Comment:
Please don't put types in docstrings in typescript files. It's not needed because the types are defined in the function and it is dangerous as it overwrites the actual return type, so if this type is wrong this is the type the IDE and I think also compiler would use. If you want to be explicit about the return type you can update to `isPDCReconcileUser = (user: TfmSessionUser): boolean =>`.

Reply:
@abhi-markan is happy for the types to stay in - so do not think we need the "whether the user is in the PDC_RECONCILE team"

Reply:
In the documentation in the codebase under `doc\docstring.md` under the TSDoc for Typescript section we explicitly omit types for the reason Ruby has mentioned above. Note the examples online at TSDocs also don't have types e.g. https://tsdoc.org/. @abhi-markan have we changed to wanting types in TS-Docs because we were following your documentation added before